compose.yml

```diff
@@ -125,34 +129,23 @@ services:
     env_file:
       - backend.env
     depends_on:
-      migrate:
-        condition: service_completed_successfully
       backend-db:
         condition: service_healthy
       init-kafka:
         condition: service_completed_successfully
     build:
       context: ./backend
       dockerfile: ../Dockerfile.backend.dev
+    entrypoint:
+      - bash
+      - "-c"
+      - "go run ./cmd/pbackend migrate && $HOME/go/bin/CompileDaemon -build='go build -buildvcs=false -o /build/pbackend ./cmd/pbackend' -command='/build/pbackend api'"
```

Comment:
the var `$HOME` results with the host's home, not the container's home, and it fails to launch.
we can change the `ENTRYPOINT` in the dockerfike instead, or using the hardcoded home of the container `/opt/app-root/src/`